### PR TITLE
Read Me

### PR DESCRIPTION
**Pull request by HubDub**
## Pull Request Description

Include the applicable information from the list below here 

Created ReadMe file that followed our definition of done. it clarified dependencies and how to download app. Moved ERD into it's own md file. 

## Steps to Test

read the code for style and errors. Does it fulfill the definition of done as you see it?

## Link to Feature Ticket

#118

### CODE REVIEW

**Review by JamsDuke — COMMENTED** [review]
Looks good

**Review by Jgregnier — COMMENTED** [review]
LGTM

**Review by csmalley89 — COMMENTED** [review]
💸

**Review by thrashMoneyVillainare — COMMENTED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+## BangazonWeb Initial Merchant Site
+
+Welcome to BangazonWeb! We want to help you sell your products to the entire world. 
+
+To install this app, you'll need to have dotnet and bower. 
+
+- If you need to install dotnet, you can do so from this [Microsoft link](https://www.microsoft.com/en-us/download/details.aspx?id=30653)
+- if you need to install bower, you can do so from this [Bower link](https://bower.io/)
+
+Copy or fork the repository on GitHub and save it on to your local machine. Navigate to the folder the bower.json resides and install the bower dependencies by typing:
```
Comment (delaineackerman) [inline README.md]: Bower.json is in the root so you could delete everything before - Install bower dependencies by typing: - in that sentence

File: README.md [hunk 2]
```diff
@@ -1 +1,33 @@
...
+
+```export NTABangazonWeb_Db_Path="/yourpath/yourpath/yourpath/bangazonWeb.db```
+
+- Windows OS:
+To set your environmental variable to the local database you'll need to type this into your Powershell terminal:
+
+`$env:NTABangazonWeb_Db_Path="/yourpath/yourpath/yourpath/bangazonWeb.db`
+
+
+After this database path has been set, create the database and set up the dotnet dependencies:
```
Comment (delaineackerman) [inline README.md]: Add: by running the following commands to this sentence.

File: README.md [hunk 3]
```diff
@@ -1 +1,33 @@
...
+
+```
+dotnet restore
+dotnet ef database update
+dotnet run
+```
```
Comment (delaineackerman) [inline README.md]: Perhaps add - the site should be accessible if installation requirements were met - or something like that.

**Review by thrashMoneyVillainare — APPROVED** [review]

**Review by delaineackerman — COMMENTED** [review]
LGTM

**Pull request merged**